inst/pkgdown/_pkgdown.yml

```diff
@@ -8,3 +8,5 @@ template:
       <img src="https://github.com/insightsengineering/hex-stickers/blob/main/PNG/nest.png?raw=true" alt="NEST" style="height: 30px;">
       <a href="https://pharmaverse.org/"><img src="https://github.com/insightsengineering/hex-stickers/blob/main/PNG/pharmaverse.png?raw=true" alt="pharmaverse" style="height: 30px;"></a>
       </div>
+  params:
+    ganalytics: "UA-125641273-1"
```

Comment:
This only works for BS3. For BS5, you need something like this: https://github.com/insightsengineering/r.pkg.template/blob/main/_pkgdown.yml#L3-L15

Reply:
Also a good idea to add consent mode: https://developers.google.com/tag-platform/devguides/consent for compliance purposes.

Reply:
```suggestion
  includes:
    in_header: |
       <script>
         // Define dataLayer and the gtag function.
         window.dataLayer = window.dataLayer || [];
         function gtag(){dataLayer.push(arguments);}
         // Default ad_storage to 'denied' as a placeholder
         // Determine actual values based on your own requirements
         gtag('consent', 'default', {
           'ad_storage': 'denied'
         });
       </script>
       <!-- Global site tag (gtag.js) - Google Analytics -->
       <script async src="https://www.googletagmanager.com/gtag/js?id=UA-125641273-1"></script>
       <script>
         window.dataLayer = window.dataLayer || [];
         function gtag(){dataLayer.push(arguments);}
         gtag('js', new Date());
         gtag('config', 'UA-125641273-1');
       </script>
```